src/shared/components/RichTable.tsx

```diff
@@ -159,16 +154,13 @@ export interface RichTableBag<D extends object = {}> {
  */
 const useRichTable = <D extends object = {}>({
   columns,
-  url,
+  get,
   pk,
```

Comment:
We can remove pk from the destructuring as it's no longer needed

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/9aa2bdd2f117930cdf0a9ce0dedbf1f7c6d186ac